Update CODEOWNERS for @kbn/typed-react-router-config

jasonrhodes opened the pull request:
Removes obs-ux-management and replaces with obx-ux-infra_services-team, based on this count of imports for this package:

```
   4 observability_ai_assistant_app
  38 apm
   1 ux
  12 profiling
   5 inventory
   5 observability_ai_assistant_management
```

Got these stats with these two steps:

```sh
ag -Q "@kbn/typed-react-router-config" --ignore="*target*" --ignore="*tsconfig.json" -c ./x-pack/plugins | tee typed-react-router-config-USAGE.log

cat typed-react-router-config-USAGE.log | cut -d "/" -f 4-4 | uniq -c
```

jasonrhodes committed:
Update CODEOWNERS for @kbn/typed-react-router-config

Removes obs-ux-management and replaces with obx-ux-infra_services-team, based on this count of imports for this package:

```
   4 observability_ai_assistant_app
  38 apm
   1 ux
  12 profiling
   5 inventory
   5 observability_ai_assistant_management
```

kibanamachine committed:
[CI] Auto-commit changed files from 'node scripts/generate codeowners'

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/247519)
* Commit: cfa7c38cb231edf379a5dab4355c5d834862733c

### Failed CI Steps
* [Quick Checks](https://buildkite.com/elastic/kibana-pull-request/builds/247519#0192e33d-e690-4456-82ce-9b0072a023fc)

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/247519","number":247519,"commit":"cfa7c38cb231edf379a5dab4355c5d834862733c"}],"number":247519}
buildkite-pr-comment-->

jasonrhodes committed:
Update kibana.jsonc

I originally tried to edit the CODEOWNERS but I forgot that's auto-generated and relies on these kibana.jsonc files. This should be the correct change now.

jasonrhodes commented:
Update: I had originally changed CODEOWNERS directly but that gets auto-generated so CI reverted it automatically based on the contents of the kibana.jsonc file in the package root dir. I've now updated the right values in that kibana.jsonc file.

jasonrhodes commented:
@elasticmachine merge upstream

elasticmachine committed:
Merge branch 'main' into update-codeowners_typed-react-router-config

kibanamachine committed:
[CI] Auto-commit changed files from 'node scripts/generate codeowners'

elasticmachine commented:
## :yellow_heart: Build succeeded, but was flaky
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/248195)
* Commit: 22651db00abc69fa8df268d29892001f108b2cab

### Failed CI Steps
* [x-pack/test_serverless/functional/test_suites/security/common_configs/config.group2.ts](https://buildkite.com/elastic/kibana-pull-request/builds/248195#0192f84f-9b68-41a0-8163-951f768621dd)
* [x-pack/test_serverless/functional/test_suites/security/common_configs/config.group5.ts](https://buildkite.com/elastic/kibana-pull-request/builds/248195#0192f84f-9b69-4be0-8369-4c627fc0f879)
* [x-pack/test_serverless/functional/test_suites/security/common_configs/config.group6.ts](https://buildkite.com/elastic/kibana-pull-request/builds/248195#0192f84f-9b6b-4721-8c5c-28a310bebc5f)
* [FTR Configs #30](https://buildkite.com/elastic/kibana-pull-request/builds/248195#0192f84f-9b9c-4e39-97b0-1d4c349dafde)
* [FTR Configs #32](https://buildkite.com/elastic/kibana-pull-request/builds/248195#0192f84f-9b9f-4b85-8fbe-f7446a4c9acc)
* [FTR Configs #60](https://buildkite.com/elastic/kibana-pull-request/builds/248195#0192f84f-9bcd-42c9-b26a-7e16bf3fab80)
* [FTR Configs #62](https://buildkite.com/elastic/kibana-pull-request/builds/248195#0192f84f-9bd0-4fd9-adb5-72bd1416f2c0)
* [FTR Configs #66](https://buildkite.com/elastic/kibana-pull-request/builds/248195#0192f84f-9bd7-43b7-901c-370f97ea640c)
* [FTR Configs #67](https://buildkite.com/elastic/kibana-pull-request/builds/248195#0192f84f-9bd9-4730-be0b-ad4906f4b6e6)
* [FTR Configs #72](https://buildkite.com/elastic/kibana-pull-request/builds/248195#0192f84f-9be2-4df8-9782-d0a4c75f1fca)
* [FTR Configs #81](https://buildkite.com/elastic/kibana-pull-request/builds/248195#0192f84f-9bf2-4975-838f-9e8d45727cd3)
* [FTR Configs #93](https://buildkite.com/elastic/kibana-pull-request/builds/248195#0192f84f-9c05-4b4b-b8d8-ada8a913dc3c)
* [FTR Configs #107](https://buildkite.com/elastic/kibana-pull-request/builds/248195#0192f84f-9c1b-4574-bb7c-98ee74f701ec)

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
> ✅ unchanged

### History
* :broken_heart: [Build #247519](https://buildkite.com/elastic/kibana-pull-request/builds/247519) failed cfa7c38cb231edf379a5dab4355c5d834862733c

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/248195","number":248195,"commit":"22651db00abc69fa8df268d29892001f108b2cab"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/247519","number":247519,"commit":"cfa7c38cb231edf379a5dab4355c5d834862733c"}],"number":248195}
buildkite-pr-comment-->
